Dockerfile

```diff
@@ -20,7 +20,7 @@ ADD package.json /
 
 # Install the node packages outlined in package-lock.json (if present) or package.json
 RUN npm install
-RUN npm install -g serverless
+RUN npm install -g serverless@2.72.2
```

Comment:
Just to keep it clean, remove the versioning "@2.72.2" from this line

Reply:
Hey Jin. I have added those lines back. Can you check now?